packages/core/src/web/bridges/hooks/use-bridge.ts

```diff
@@ -20,7 +20,7 @@ const DEFAULT_VALUE_FOR_MCP_APP_BRIDGE: BridgeInterface = {
       left: 0,
     },
   },
-  maxHeight: window.innerHeight,
+  maxHeight: typeof window !== "undefined" ? window.innerHeight : 400,
```

Comment:
```suggestion
  maxHeight: window?.innerHeight ?? 400,
```

Reply:
<img width="316" height="58" alt="image" src="https://github.com/user-attachments/assets/36264271-34d1-4d0d-979c-79a303558938" />

However I get this when I run the dev environment of vite

Reply:
But that's because I'm setting a constant with a window default value. As vite is evaluating the file for ssr, then it tries to access window which indeed not defined (no declaration AT all).

A solution is to put the default value getter into a function